### PR TITLE
enh: add some more documentation for antivirus usage

### PR DESCRIPTION
**Pull request by max-nextcloud**
grafik

### CODE REVIEW

**Comment by max-nextcloud** [comment]
https://github.com/nextcloud/files_antivirus/pull/284 added the commands.

File: admin_manual/configuration_server/antivirus_configuration.rst [hunk 1]
```diff
@@ -169,10 +169,55 @@ Executable
...
+
+The background scanner does not require any manual intervention.
+However at times you might want to inspect it or perform tasks on it.
+
+Get info about files in the scan queue
+~~~~~~~~
+
+::
+
+    occ files_antvirus:status [-v]
```
Comment (luka-nextcloud) [inline admin_manual/configuration_server/antivirus_configuration.rst:192]: Type: should be `occ files_antivirus:status [-v]` (missing `i`) ;)

File: admin_manual/configuration_server/antivirus_configuration.rst [hunk 2]
```diff
@@ -169,10 +169,55 @@ Executable
...
+::
+
+    occ files_antivirus:background-scan [-v] [-m MAX]
+
+Manually scan a single file
+~~~~~~~~
+
+::
+
+    occ files_antvirus:scan <path>
```
Comment (luka-nextcloud) [inline admin_manual/configuration_server/antivirus_configuration.rst:207]: Type: should be `occ files_antivirus:scan <path>` (missing `i`) ;)

**Comment by kesselb** [comment]
Hi, thanks for your nice pull request :+1: 

We have a different page for most of the occ commands: https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/occ_command.rst

Do you think it makes sense to move them over there?

**Comment by max-nextcloud** [comment]
I tried to include all feedback:
* typos fixed
* added occ commands also on the occ command page

I also changed the occ commands in the antivirus docs to include `sudo -u www-data php ` to match the ones in the occ docs.

@luka-nextcloud @kesselb could you have another look?

**Review by icewind1991 — APPROVED** [review]

**Pull request merged**